docs/extensions/logging-and-monitoring.md

```diff
@@ -1,32 +1,39 @@
-# Logging and Monitoring configuration
+# Logging and Monitoring for Extensions
 
-By default, Gardener deploys a central Prometheus, AlertManager, and Grafana instance into the `garden` namespace of all seed clusters.
-Additionally, as part of the shoot reconciliation flow, it deploys a shoot-specific Prometheus, Grafana (and, if configured, an AlertManager) into the shoot namespace next to the other control plane components.
+Gardener provides an integrated logging and monitoring stack for alerting, monitoring and troubleshooting of its managed components by operators or end users. For further information how to make use of it in these roles, refer to the corresponding guides for [exploring logs](https://github.com/gardener/logging/tree/master/docs/usage/logging.md) and for [monitoring with Grafana](https://grafana.com/docs/grafana/latest/getting-started/getting-started/#all-users).
 
-Configurable by the `Logging` feature gate in the `gardenlet` configuration it might also deploy a central fluent-bit and Loki Statefulset into the `garden` namespace of all seed clusters.
-Additionally, as part of the shoot reconciliation flow, it might deploy a shoot-specific Loki into the shoot namespace next to the other control plane components.
+The components that constitute the logging and monitoring stack are managed by Gardener. By default, it deploys [Prometheus](https://prometheus.io/), [Alertmanager](https://prometheus.io/docs/alerting/latest/alertmanager/) and [Grafana](https://grafana.com/) into the `garden` namespace of all seed clusters. If the `Logging` [feature gate](https://github.com/gardener/gardener/blob/master/docs/deployment/feature_gates.md) in the `gardenlet` configuration is enabled, it will deploy also [fluent-bit](https://fluentbit.io/) and [Loki](https://grafana.com/oss/loki/) there too.
```

Comment:
Also and too are synonyms, change the end of the paragraph to:
it will also deploy [fluent-bit](https://fluentbit.io/) and [Loki](https://grafana.com/oss/loki/).